### PR TITLE
[Docs][SIEM]Adds prebuilt rule descriptions

### PR DESCRIPTION
**Pull request by benskelker**
Adds prebuilt rule descriptions to the SIEM docs. This is generated from the JSON rule files, with only minor edits.

[Preview](http://stack-docs_831.docs-preview.app.elstc.co/guide/en/siem/guide/master/prebuilt-rules.html)

### CODE REVIEW

File: docs/en/siem/rule-details/telnet-port-activity.asciidoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
+[[telnet-port-activity]]
+=== Telnet Port Activity
+
+Detects network events that may indicate the use of Telnet traffic.
+Telnet is commonly used by system administrators to remotely control older or
+embed ed systems using the command line shell. It should almost never be
```
Comment (randomuserid) [inline docs/en/siem/rule-details/telnet-port-activity.asciidoc]: embedded

File: docs/en/siem/rule-details/trusted-developer-application-usage.asciidoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
+[[trusted-developer-application-usage]]
+=== Trusted Developer Application Usage
+
+Identifies possibly suspicious activity using trusted Windows developer
+activity.
```
Comment (randomuserid) [inline docs/en/siem/rule-details/trusted-developer-application-usage.asciidoc]: trusted Windows developer utility program activity

File: docs/en/siem/rule-details/unusual-parent-child-relationship-.asciidoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
+[[unusual-parent-child-relationship-]]
+=== Unusual Parent-Child Relationship 
+
+Identifies Windows programs run from unexpected parent processes. This could
```
Comment (randomuserid) [inline docs/en/siem/rule-details/unusual-parent-child-relationship-.asciidoc]: run by unexpected parent processes

File: docs/en/siem/rule-details/trusted-developer-application-usage.asciidoc (hunk 2 above)
Comment (benskelker) [reply docs/en/siem/rule-details/trusted-developer-application-usage.asciidoc]: @randomuserid 
Thanks for noticing this. We need to improve it a bit more:

`Identifies possibly suspicious activity using trusted Windows developer utility program activity.`

Should it be:
`Identifies possibly suspicious activity using a trusted Windows developer utility program.`?

Comment (randomuserid) [reply docs/en/siem/rule-details/trusted-developer-application-usage.asciidoc]: yes that is better

**Comment by benskelker** [comment]
Made all corrections - thanks

**Pull request merged**